t/plugin/limit-conn.t

```diff
@@ -1328,3 +1328,47 @@ GET /t
 {"error_msg":"failed to check the configuration of plugin limit-conn err: property \"allow_degradation\" validation failed: wrong type: expected boolean, got string"}
 --- no_error_log
 [error]
+
+
+
+=== TEST 35: invalid route: wrong rejected_msg
```

Comment:
I just notice we have `t/plugin/limit-conn2.t`. Let's add test there.

Reply:
okay